### PR TITLE
PAM library code cleanup

### PR DESCRIPTION
**Pull request by dakshinai**
Changes to PAM library code to log PAM conversation messages. Passed hacking.sh.

### CODE REVIEW

**Comment by eliqiao** [comment]
@dakshinai We have a Docker file which was used by travis-ci, in that job , we did the build and unit test case checkin, I think we can add hacking check in to that Docker file, so ci will cover static checking too.

File: lib/pam/pam.go [hunk 1]
```diff
@@ -2,9 +2,9 @@ package pam
 
 import (
 	"errors"
-	"fmt"
 	"github.com/intel/rmd/lib/pam/config"
 	"github.com/msteinert/pam"
+	"github.com/sirupsen/logrus"
```
Comment (eliqiao) [inline lib/pam/pam.go]: How about using log "github.com/sirupsen/logrus" just like other model does.

Comment (dakshinai) [reply lib/pam/pam.go]: Looks better. I have made the changes to the PR.

**Comment by dakshinai** [comment]
Changes done.

**Pull request merged**